packages/react-sdk/src/css/css.ts

```diff
@@ -69,25 +72,31 @@ export const generateCss = ({
 }: CssConfig) => {
   const globalSheet = createRegularStyleSheet({ name: "ssr" });
   const sheet = createRegularStyleSheet({ name: "ssr" });
-  const parentIdByInstanceId = new Map<Instance["id"], Instance["id"]>();
-  for (const instance of instances.values()) {
-    for (const child of instance.children) {
-      if (child.type === "id") {
-        parentIdByInstanceId.set(child.value, instance.id);
-      }
-    }
-  }
 
-  const descendantSelectorByInstanceId = new Map<Instance["id"], string>();
-  for (const prop of props.values()) {
-    if (prop.name === "selector" && prop.type === "string") {
-      descendantSelectorByInstanceId.set(prop.instanceId, prop.value);
+  addGlobalRules(globalSheet, { assets, assetBaseUrl });
+  globalSheet.addMediaRule("presets");
+  const presetClasses = new Map<Instance["component"], string>();
+  const scope = createScope([], (name) => name);
+  for (const [component, meta] of componentMetas) {
+    const [_namespace, componentName] = parseComponentName(component);
+    const className = `ws-p-${scope.getName(component, kebabCase(componentName))}`;
```

Comment:
do we really need the "-p" or could it be just ws-name?

Reply:
Can conflict with instance styles. I am planning to use `ws-i-*`. Alternatively would be deduplicated like this `ws-link ws-link_1` where first is preset and the second is local.